theano/tensor/elemwise.py

```diff
@@ -504,7 +504,7 @@ def __setstate__(self, d):
         self.nfunc = None
         self.inplace_pattern = frozendict(self.inplace_pattern)
         if getattr(self, 'nfunc_spec', None):
-            self.nfunc = getattr(numpy, self.nfunc_spec[0])
+            self.nfunc = getattr(np, self.nfunc_spec[0])
         elif 0 < self.scalar_op.nin < 32:
             self.ufunc = np.frompyfunc(self.scalar_op.impl,
                                           self.scalar_op.nin,
```

Comment:
@Sentient07 the conversations are [here](https://github.com/Theano/Theano/pull/5640). 

>Four or five files are conflicting often. It was hard for me to fix the files. I am learning now. So, the commit messages cannot be clear and it is messy. I guess it will be a headache for reviewing this PR. But, now I know how and where to do changes. In the weekend I will do a fresh fork and do fix the files one by one. That will be clear and easy.